www/build.sh

```diff
@@ -81,17 +82,22 @@ else
   ls | grep "roc_nightly.*tar.gz" | xargs rm
   # simplify dir name
   mv roc_nightly* roc_nightly
+  roc='./roc_nightly/roc'
 
   echo 'Building tutorial.html from tutorial.md...'
   mkdir www/build/tutorial
-
-  ./roc_nightly/roc version
-  ./roc_nightly/roc run www/generate_tutorial/src/tutorial.roc -- www/generate_tutorial/src/input/ www/build/tutorial/
-
-  # cleanup
-  rm -rf roc_nightly roc_releases.json
 fi
 
+$roc version
+$roc run www/generate_tutorial/src/tutorial.roc -- www/generate_tutorial/src/input/ www/build/tutorial/
 mv www/build/tutorial/tutorial.html www/build/tutorial/index.html
 
+# for new wip site
+mkdir www/build/wip
+roc run www/wip_new_website/main.roc -- www/wip_new_website/content/ www/build/wip
```

Comment:
@lukewilliamboswell, I moved the build steps to this script, I really wanted to use arguments with the roc build script but couldn't due to https://github.com/roc-lang/basic-cli/issues/82